### PR TITLE
Add "Schleifen" exercise

### PR DESCRIPTION
**Pull request by akalcik**

### CODE REVIEW

**Comment by akalcik** [comment]
[Übung C#: Schleifen](https://trello.com/c/taxDIOeI/42-%C3%BCbung-c-schleifen)

File: csharp/Schleifen/Readme.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Es gibt auch eine Fußgesteuerte Variante einer ```while``` Schleife. Die siete so aus:
+
+```csharp
+int anzahlVonZeichen = 0;
+do
+{
+    Console.Write("X");
+    anzahlVonZeichen++;
+}
+while (anzahlVonZeichen < 10)
```
Comment (rigerta) [inline csharp/Schleifen/Readme.md]: **;** missing in the end of the do...while()

Comment (akalcik) [reply csharp/Schleifen/Readme.md]: Oh you are right. I'll repair it.

**Comment by kwaxi** [comment]
Proposed some changes in a branch to this branch.

**Pull request merged**